Core/Relationships/RelationshipResolver.cs

```diff
@@ -124,7 +124,9 @@ public RelationshipResolver(RelationshipResolverOptions options, IRegistryQuerie
         }
 
         /// <summary>
-        /// Attempts to convert the module_names to ckan modules via  CkanModule.FromIDandVersion and then calls RelationshipResolver.ctor(IEnumerable{CkanModule}, Registry, KSPVersion)"/>
+        /// Attempts to convert the identifiers to CkanModules and then calls RelationshipResolver.ctor(IEnumerable{CkanModule}, IEnumerable{CkanModule}, Registry, KSPVersion)"/>
+        /// For modulesToInstall CkanModule.FromIDandVersion is used, which tries to find the latest compatible version,
+        /// for modulesToRemove Registry.InstalledModule is used to get the currently installed version.
         /// </summary>
         /// <param name="modulesToInstall">Identifiers of modules to install</param>
```

Comment:
Elements of `modulesToInstall` aren't just identifiers, they can also be `identifier=version`. That's why `FromIDAndVersion` exists. We should have added `///<param>` tags explaining this in `FromIDandVersion` a long time ago. Can we now?

Reply:
Tried to imrpove the code docs situation a bit. Feel free to push some changes if something doesn't make sense or you're able to formulate it in a more understandable way.